index.html

```diff
@@ -11,7 +11,7 @@
 			content="A smart shopping list that learns your purchase habits and makes suggestions, so you don't forget to buy what's important."
 		/>
 		<link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
-		<meta name="theme-color" content="#000000" />
```

Comment:
The `theme-color` meta tag is a nice-to-have that _mostly_ relates to installed progressive web apps. Since it's not really being focused on, I opted to replace it.

- [`theme-color` meta tag (MDN)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color)